docs/user-guides/installing-perses-in-a-container.md

```diff
@@ -28,39 +27,36 @@ $ docker run --name perses -d -p 127.0.0.1:8080:8080 persesdev/perses
 The details in this command are that we give the container a referencable name (--name perses), detach the container
 from the command line (-d), map the local machine port 8080 to the container port 8080 (-p 127.0.0.1:8080:8080), and use
 the image version supported in these instructions (latest). Note: you can use any local port you have available, but you
-need to map to the container port 8080.
+need to map to container port 8080.
 
-### Note: What to do with failures?
+### Tags
 
-Don't worry if at anytime in these instructions you encounter failures during installation, testing, data population, or build
-results. The container can be rerun anytime after you fix any problems reported. You might have to remove the perses
-container depending on how far you get before something goes wrong. Just stop, remove, and restart it:
+We are publishing a set of tags to match different requirements when running Perses in a container:
 
-```shell
-# podman command.
-#
-$ podman container stop perses
+1. Our images can run either on **arm64** or on **amd64** platform.
+2. We have two different types of image: `distroless` and `distroless-debug`.
+   The first one is meant to be used in production as it doesn't contain anything excepting the Perses binaries.
+   The second one contains a shell and should be used to debug any issue you might encounter with your container.
+3. The tag `latest` is an alias for the `distroless` distribution and is erased with the latest release.
+4. The tag `latest-debug` is an alias for the `distroless-debug` distribution and is erased with the latest release.
+5. We are also providing an image for each commit created on the `main` branch.
+   The tag created for each commit is following this pattern: `main-<YYYY-MM-DD>-<short commit ID>-<distribution>`
 
-$ podman container rm perses
+Examples:
 
-$ podman run --name perses -d -p 127.0.0.1:8080:8080 persesdev/perses
-
-# docker command.
-#
-$ docker container stop perses
-
-$ docker container rm perses 
-
-$ docker run --name perses -d -p 127.0.0.1:8080:8080 persesdev/perses
-```
+- images built from the main branch:  `persesdev/perses:main-2023-12-14-f66e10ce-distroless-debug`
+  or `persesdev/perses:main-2023-12-14-f66e10ce-distroless`
+- latest Perses image: `persesdev/perses` or `persesdev/perses:latest-debug`
+- precise release image: `persesdev/perses:v0.42.1`, `persesdev/perses:v0.42.1-debug`, `persesdev/perses:v0.42.1-distroless`, `persesdev/perses:v0.42.1-distroless-debug`
+- minor release image: `persesdev/perses:v0.42`, `persesdev/perses:v0.42-debug`, `persesdev/perses:v0.42-distroless`, `persesdev/perses:v0.42-distroless-debug`
 
 ## Connect a browser (default)
 
 Open the Perses console at http://localhost:8080.
```

Comment:
```suggestion
Open the Perses page at http://localhost:8080.
```
no?

Reply:
E.g Openshift also comes with the term console for its UI, but yes I also think that another term like page/website/UI would sound better here.